src/i3ipc.ml

```diff
@@ -249,6 +271,23 @@ module Reply = struct
 
   let result_of_command_outcome { success; error } =
     if success then Result.Ok () else Result.Error (error |? "")
+
+  type binding_modes = string list
+    [@@deriving of_yojson, show]
+
+  type config = {
+    config : string
+  } [@@deriving of_yojson, show]
+
+  let pp_config fmt config =
+    Format.pp_print_string fmt config
+
+  type tick = {
+    tick_success : bool [@key "success"]
+  } [@@deriving of_yojson, show]
```

Comment:
Hmm, for this one I guess we can assume it only contain the boolean. So this is fine. But then, we don't need to expose the record in the public API.